apache-nifi.yaml

```diff
@@ -46,43 +51,86 @@ pipeline:
       expected-commit: 338083f6850b61cd2651e41bbd73b3cb5330d98c
 
   - runs: |
+      # Build NiFi
       ./mvnw -T$(nproc)C -q package \
         --no-snapshot-updates \
         --no-transfer-progress \
         --fail-fast \
         -DskipTests \
         -Dfrontend.skipTests=true \
         -Drat.skip=true
-      mkdir -p ${{targets.destdir}}/usr/share/
-      cp -ar nifi-assembly/target/nifi-${{package.version}}-bin/nifi-${{package.version}} ${{targets.destdir}}/usr/share/nifi
 
-  - runs: |
-      for dir in conf database_repository flowfile_repository content_repository provenance_repository state; do
-        mkdir -p "${{targets.destdir}}/usr/share/nifi/${dir}"
-      done
+      # Install NiFi
+      mkdir -p "${{targets.contextdir}}/${NIFI_BASE}"
+      cp -ar "nifi-assembly/target/nifi-${{package.version}}-bin/nifi-${{package.version}}" "${{targets.contextdir}}/${NIFI_HOME}"
 
+      # Create expected directories
       for dir in conf database_repository flowfile_repository content_repository provenance_repository state; do
-        mkdir -p ${{targets.destdir}}/usr/share/nifi/logs/${dir}
+        mkdir -p "${{targets.contextdir}}/${NIFI_HOME}/${dir}"
```

Comment:
we should have one additional dir for logs: `${NIFI_HOME}/logs`: https://github.com/apache/nifi/blob/338083f6850b61cd2651e41bbd73b3cb5330d98c/nifi-docker/dockerhub/Dockerfile#L77